tests/datasets/test_datadoc_metadata.py

```diff
@@ -499,7 +499,7 @@ def test_merge_extracted_and_existing_dataset_metadata(metadata_merged: Datadoc)
         ), f"{field} in merged metadata did not match existing metadata"
 
     # Special cases
-    assert metadata_merged.dataset.version_description is None
+    assert metadata_merged.dataset.version_description is not None
```

Comment:
```suggestion
```

We can just remove this, the field is tested in the `for` loop on L495 and it's not a special case any longer